app/assets/stylesheets/components/_mobile-filters.scss

```diff
@@ -86,10 +87,14 @@
       }
     }
 
+    .facets__content {
+      padding-top: govuk-spacing(2);
```

Comment:
I think there's an alternative solution here. Each of the child elements (either an expander or an option select) has a slightly odd thing where they've got a padding and margin bottom of 10px. That means each one is reliant on the one above to position it correctly within the dividing lines, with the addition of this 10px at the very top to position the first one.

This feels a bit clunky to me - I think it would be cleaner if there was no margin and all these child elements had 10px of padding top and bottom. Thoughts?

Reply:
I would agree, however it's messy because some search results filters have weird things like this `facets-container` that contains a `govuk-form-group` as the first item:

<img width="488" alt="image" src="https://github.com/user-attachments/assets/6c347c86-ac2e-4559-9afe-60826cbb5f09">

If we used your suggestion, we'd still need to add an extra 10px of top padding to the `facets-container` otherwise the first radio button is flush with the first border. Given that my solution solves both problems, albeit in a slightly hacky way, I think it's the better option right now.

In all honesty this entire module needs a complete overhaul: it is incredibly confusing and comprises different components that don't play well together. Another one for the refactoring backlog...?

Reply:
Personally I'd go with that as a solution but as this whole section is going to be rebuilt shortly I'm happy to leave as-is.